letsencrypt/cli.py

```diff
@@ -638,7 +669,7 @@ def run(config, plugins):  # pylint: disable=too-many-branches,too-many-locals
     _suggest_donation_if_appropriate(config)
 
 
-def obtain_cert(config, plugins):
+def obtain_cert(config, plugins, lineage=None):
```

Comment:
this doesn't seem like it can a) be called without a lineage supplied, or b) called with a lineage of None, as the only place this is called is on line 912 inside of renew.
at worst this is sloppy looking, but not actually harmful